src/toggle_button.rs

```diff
@@ -114,6 +118,16 @@ impl<'a> ToggleButton<'a> {
         self.smartstate.set(smartstate);
         self
     }
+
+    /// provides a minimum width for the widget
+    ///
+    /// if the width calculated from the contents plus padding plus border are
+    /// less than the provided minimum width the width of the widget will be increased
+    pub fn with_min_width(mut self, width: u32) -> Self {
+        self.min_width = Some(width);
+        self.is_modified = true;
+        self
+    }
```

Comment:
```suggestion
    pub fn with_min_width(mut self, width: u32) -> Self {
        self.min_width = Some(width);
        self
    }
```

As said in the more detailed comment, this should be handled by the smartstate.